src/pages/dataTableUtilities/getPageInfoColumns.js

```diff
@@ -59,9 +59,47 @@ const PER_PAGE_INFO_COLUMNS = {
     ['entryDate', 'confirmDate', 'transactionCategory'],
     ['enteredBy', 'otherParty'],
   ],
+  requisitionItemDetail: [['openVialWastage', 'closedVialWastage']],
+  breach: [
+    [
+      'breachTemperatureRange',
+      'numberOfAffectedBatches',
+      'breachDuration',
+      'location',
+      'affectedQuantity',
+    ],
+  ],
 };
 
 const PAGE_INFO_ROWS = (pageObject, dispatch, route) => ({
+  openVialWastage: {
+    title: `${pageInfoStrings.open_vial_wastage}:`,
+    info: pageObject.openVialWastage(pageObject.lastRequisitionDate),
+  },
```

Comment:
this will be an `Item` which will have these methods as shown in this one:

![image](https://user-images.githubusercontent.com/35858975/77726174-2c414a80-705c-11ea-890b-dac060efa5ae.png)